slicingTool/runProjectSlicingTool.sh

```diff
@@ -26,14 +34,23 @@ if [[ -n $DB && -n $USER && -n $PASS ]]
 then
     if [[ $USER =~ "curator" ]]
     then
-	echo Dropping $DB ...
-	mysql -u$USER -p$PASS -e "drop database if exists $DB"
-#	mysql -u$USER -p$PASS -e "drop database if exists ${DB}_myisam"
-#	mysql -u$USER -p$PASS -e "create database ${DB}_myisam"
-#	mysql -u$USER -p$PASS -e "drop database if exists gk_central"
-#	mysql -u$USER -p$PASS -e "create database gk_central"
+        # Attempts to find database $DB and if successful (mysqlshow returns a zero exit status) it is backed up
+        # before being dropped
+        mysqlshow $DB > /dev/null 2>&1
+        if [[ $? == 0 ]]
+        then
+            echo Backing up $DB ...
+            mysqldump -u$USER -p$PASS $DB > $DB.dump
+        fi
```

Comment:
Is an `else` block needed to let the user notify that `mysqlshow` failed? _IF_ it fails, is that not a problem??

Reply:
Yeah, that's a good point.  I'll add an `else` block.